package.json

```diff
@@ -19,17 +27,24 @@
     "mocha-multi-reporters": "1.5.1",
     "mochawesome": "6.2.2",
     "nyc": "15.1.0",
-    "remarkable": "2.0.1"
+    "remarkable": "2.0.1",
+    "rollup": "^2.75.3",
+    "rollup-plugin-babel": "^4.4.0",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-uglify": "^6.0.4"
```

Comment:
Are we using the right plugins? At least looking at the `rollup-plugin-babel` source, it seems to have been deprecated in favor of `@rollup/plugin-babel`.

- https://www.npmjs.com/package/rollup-plugin-babel
- https://github.com/rollup/rollup-plugin-babel

Also noticed in the `yarn.lock` file some duplication of transitive dependencies (but different versions) around babel being pulled in. Let's double check and make sure we're not using deprecated plugins and add required/compatible version of `@babel/core` needed by rollup.

Reply:
Updated with new plugins.